tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -499,10 +499,18 @@ TEXT_DRAW_TEST_P(Fonts, TestFonts) {
     __DrawLoremIpsum(context, path.get(), keys, values);
 }
 
+// Segoe UI is noticeably different on ARM, but we still want to test other fonts so only remove Segoe
+#ifdef WINOBJC_DISABLE_TESTS
```

Comment:
We don't need to test every font with every API.  We use DWrite under the covers, and have to trust it.  So, just remove Segoe UI from this list.  The fonts could sometimes be different on the phone either to adjust for the form factor or size or both.  

---
In reply to: [112273877](https://github.com/Microsoft/WinObjC/pull/2566#discussion_r112273877) [](ancestors = 112273877)